src/pages/{Page.slug}.tsx

```diff
@@ -31,7 +45,7 @@ export const query = graphql`
       description
       image {
         id
-        gatsbyImageData
+        url
       }
```

Comment:
I _think_ this was just a copy pasta problem from before. But from looking at contentful, these `Page` images are just `id` and `url`.